Sources/Property.swift

```diff
@@ -419,16 +420,27 @@ public final class Property<Value>: PropertyProtocol {
 
 	/// Initializes an existential property which wraps the given property.
 	///
+	/// - note: The resulting property retains the given property.
+	///
 	/// - parameters:
 	///   - property: A property to be wrapped.
 	public init<P: PropertyProtocol>(_ property: P) where P.Value == Value {
-		sources = Property.capture(property)
 		disposable = nil
 		_value = { property.value }
 		_producer = { property.producer }
 		_signal = { property.signal }
 	}
 
+	/// Initializes a composed property which reflects the given property.
+	///
+	/// - note: The resulting property does not retain the given property.
+	///
+	/// - parameters:
+	///   - property: A property to be wrapped.
+	public convenience init<P: PropertyProtocol>(reflecting property: P) where P.Value == Value {
```

Comment:
Just wanted to note that `reflecting` doesn't feel quite right to me, though I haven't been able to come up with a suitable alternative yet.